resources/user.go

```diff
@@ -224,6 +244,70 @@ type UserUID struct {
 	name string
 }
 
+// UserResAutoEdges holds the state of the auto edge generator.
+type UserResAutoEdges struct {
+	UIDs    []ResUID
+	pointer int
+}
+
+// AutoEdges returns the the user's group.
```

Comment:
Technically this comment is wrong. Can you spiffy it up ever so slightly? In the best case scenario it should explain the logic of what it's doing and why, but I won't force that on you if you're really not in the mood.